scripts/p2p-test-tool.py

```diff
@@ -0,0 +1,403 @@
+                    notices['fail'].append(f"{container.name}: Errors found in node logs.")
+        if test == "count":
+            for container in client.containers.list(all=True, filters={"name":f"peer\d.{args.network}"}):
+                if test_node_logs_for_correct_peers_count(container) == 0:
+                    notices['pass'].append(f"{container.name}: Peers count correct in node logs.")
+                else:
+                    notices['fail'].append(f"{container.name}: Peers count incorrect in node logs.")
+        if test == "eval":
+            for container in client.containers.list(filters={"name":f"peer\d.{args.network}"}):
+                if test_node_eval_of_rholang_files(container) == 0:
+                    notices['pass'].append(f"{container.name}: Rholang evaluation of files performed correctly.")
+                else:
+                    notices['fail'].append(f"{container.name}: Rholang evaluation of files failed.")
+        if test == "repl":
+            # for container in client.containers.list(all=True, filters={"name":"peer\d\.rchain.coop"}): # this will enable all peers to be checked
+            # for container in client.containers.list(all=True, filters={"name":f"{args.grpc_host}"}):
+            for container in client.containers.list(all=True, filters={"name":f"peer0.{args.network}"}):
+                if test_repl_load(container) == 0:
+                    notices['pass'].append(f"{container.name}: REPL loader success!")
+                else:
+                    notices['fail'].append(f"{container.name}: REPL loader failure!")
+
+    print("===========================================================")
+    print("=================TEST SUMMARY RESULTS======================")
+    if notices['pass']:
+        for notice in notices['pass']:
+            print(f"PASS: {notice}")
+    if notices['fail']:
+        for notice in notices['fail']:
+            print(f"FAIL: {notice}")
+        print('FAIL: Part or all of tests failed in one or more peer nodes.')
+        raise Exception('FAIL: Part or all of tests failed in one or more peer nodes.')
+    else:
+        print("PASS ALL: All tests successfully passed")
+    print("===========================================================")
+    print("===========================================================")
+
+
+def deploy_demo():
+    cmd = f"{RNODE_CMD} --deploy-demo"
+    for container in client.containers.list(all=True, filters={"name":f"peer\d.{args.network}"}):
+        try:
+            r = container.exec_run(cmd=cmd, detach=True)
+        except Exception as e:
+            print(e)
+
+def test_node_eval_of_rholang_files(container):
+    print(container.name)
+    cmd = f"ls /usr/share/rnode/examples/*.rho"
+    r = container.exec_run(['sh', '-c', cmd])
+    for file_path in r.output.decode('utf-8').splitlines():
+        print(file_path)
+        eval_r = container.exec_run(['sh', '-c', f'{RNODE_CMD} --eval {file_path}'])
+        #print(eval_r)
```

Comment:
Unused comment?

Reply:
Yes, sometimes this can be helpful if you want to look at output it just gets to be a lot. Let me fix it up.